server/db/index.js

```diff
@@ -9,9 +9,29 @@ const Genres = require('./Genre');
 const Payments = require('./Payment');
 
 
-// put assoiations here
 
-//put assiations here
+// User Associations
+Users.hasMany( Orders );
+Users.hasMany( Payments );
+Users.hasMany( Reviews );
+
+
+//Review Association
+ Reviews.belongsTo( Albums );
+ Reviews.belongsTo( Songs );
+ Reviews.belongsTo(Users, {through: 'userReview'} ) ;
```

Comment:
I would like to see you do both ways. SO if a user hasMany orders, what does an order have? Try to look at 
http://docs.sequelizejs.com/en/latest/docs/associations/